### PR TITLE
Tweets

### PR DESCRIPTION
**Pull request by kma3a**

### CODE REVIEW

**Comment by kma3a** [comment]
@mattbaker @mikelikesbikes can you give us some feedback on the tests?

File: TwitterStories/app/controllers/tweets_controller.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+class TweetsController < ApplicationController
+  def new
+    @tweet = Tweet.new
+  end
+
+  def create
+    @tweet = Tweet.new(tweet_params)
+    if @tweet.save
```
Comment (mattbaker) [inline TwitterStories/app/controllers/tweets_controller.rb:8]: Should probably assert that the tweet gets saved, or call `#create` and check `#valid?` to see if something went wrong. Your currents tests sort of implicitly assert that the tweet gets saved by checking that it redirects to a path using the tweet's ID.

File: TwitterStories/spec/controllers/tweets_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        .should_receive(:new)
+        .and_return(tweet)
+      post :new
+      expect(assigns(:tweet)).to eq tweet
+    end
+  end
+
+  describe "#create" do
+    subject { post :create, :tweet => { :text => "Poo", :user_id => 1}}
+
```
Comment (mikelikesbikes) [inline TwitterStories/spec/controllers/tweets_controller_spec.rb:18]: You're missing a test that the tweet gets saved to the DB

**Comment by mikelikesbikes** [comment]
You should remove files that you aren't using (that were created by the generator that you aren't supposed to be using). For example: `tweet_helper.rb`, `tweet_helper_spec.rb`, `tweets.js.coffee`, etc.

:trollface:

**Pull request merged**